localstack/utils/cloudformation/template_deployer.py

```diff
@@ -258,6 +258,47 @@ def replace(params, **kwargs):
                 'Name', 'Type', 'Value', 'Description', 'AllowedPattern', 'Policies', 'Tier'))
         }
     },
+    'SecretsManager::Secret': {
+        'create': {
+            'function': 'create_secret',
+            'parameters': {
+                'Name': 'Name',
```

Comment:
Can we please also map the remaining properties - 'Description', 'KmsKeyId', 'SecretString', 'Tags': https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-secretsmanager-secret.html#aws-resource-secretsmanager-secret-syntax.json . We can probably use `select_parameters(..)` here, like in line 278 below.